spec/support/factory_bot.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
```

Comment:
Just 1 question here. Where do you use factorybot gem, or why you added it?

Reply:
The gem was previously added in this PR: [ Feature#16 create test data](https://github.com/BC-MAR-21-ROR/kata-07-rejoj-checador-equipo-03-k07/pull/21)

Reply:
oh gotcha